halo/app/app_config.go

```diff
@@ -78,6 +92,7 @@ var (
 		attesttypes.ModuleName,
 		valsynctypes.ModuleName, // Wraps staking module end blocker (must come after attest module)
 		upgradetypes.ModuleName,
+		evmstaking2types.ModuleName,
```

Comment:
you shouldn't add this modile if feature is disabled. Also, the ordering isn't correct

Reply:
remember to run normal tests to ensure they still work (regression)